qiskit_experiments/calibration_management/__init__.py

```diff
@@ -129,52 +144,9 @@
     xp = cals.get_template("xp")
 
 which would return the default xp schedule block template for all qubits.
-
-.. currentmodule:: qiskit_experiments.calibration
-
-.. autosummary::
-    :toctree: ../stubs/
-
-    BackendCalibrations
-    Calibrations
-    Frequency
-    Amplitude
-    Drag
-
-Calibration experiments
-=======================
-
-The following experiments are designed to calibrate parameter values. Some experiments such
-as :class:`QubitSpectroscopy` can both be seen as characterization and calibrations
-experiments. Such experiments can be found in the :mod:`qiskit_experiments.characterization`
-module.
-
-.. autosummary::
-    :toctree: ../stubs/
-
-    DragCal
-    Rabi
-    FineAmplitude
-
-Calibration analysis
-====================
-.. autosummary::
-    :toctree: ../stubs/
-
-    OscillationAnalysis
-    DragCalAnalysis
-    FineAmplitudeAnalysis
-
 """
 
-from .drag import DragCal
-from .rabi import Rabi, EFRabi
-from .fine_amplitude import FineAmplitude
-
-from .analysis.oscillation_analysis import OscillationAnalysis
-from .analysis.drag_analysis import DragCalAnalysis
-from .analysis.fine_amplitude_analysis import FineAmplitudeAnalysis
+from .calibrations import Calibrations
```

Comment:
During the past review discussion, I found some people still prefer absolute import path and some packages still use absolute, while others use relative path. Can you define a policy for import path? If it's allowed to mix them this is not a big issue.

Reply:
I don't have a really strong opinion. If its within a self contained module for init (ie calibration_management in this case) i think relative imports are ok so you dont get super long file lines.